doc/build/changelog/unreleased_20/9870.rst

```diff
@@ -0,0 +1,7 @@
+.. change::
+    :tags: bug, orm, regression
+    :tickets: 9870
+
+    Fixed regression in the 2.0 series where a query that used
+    :func:`.undefer_group` with :func:`_orm.selectinload` or
+    :func:`_orm.subqueryload` would raise an ``AttributeError``.
```

Comment:
Feel free to add "Pull request courtesy of Matthew Martin"

Reply:
Added.